modules/lightning_features/lightning_core/modules/lightning_search/css/entity-search.css

```diff
@@ -0,0 +1,3 @@
+.ui-autocomplete-loading[data-entity-search] {
```

Comment:
Just tried to inspect the form to see if a class was different and I got a PluginException:


    Drupal\Component\Plugin\Exception\PluginException: Plugin (entity_view) instance class "Drupal\lightning_core\Plugin\Block\EntityViewBlock" does not exist. in Drupal\Component\Plugin\Factory\DefaultFactory::getPluginClass() (line 97 of /Users/adam.balsam/Sites/balsama-lightning/docroot/core/lib/Drupal/Component/Plugin/Factory/DefaultFactory.php).